beacon-chain/monitor/service.go

```diff
@@ -236,6 +236,13 @@ func (s *Service) monitorRoutine(stateChannel chan *feed.Event, stateSub event.S
 				} else {
 					s.processAggregatedAttestation(s.ctx, data.Attestation)
 				}
+			case operation.SingleAttReceived:
+				data, ok := e.Data.(*operation.SingleAttReceivedData)
+				if !ok {
+					log.Error("Event feed data is not of type *operation.SingleAttReceivedData")
```

Comment:
i wonder why we don't print the Type of the data here for old ones, maybe that would be useful

Reply:
is it because we already do that once in the marshal reader?

Reply:
Are you sure you are looking at the correct file? I don't see any marshal reader in the monitor

Reply:
oops then maybe we should print the type somewhere.